installer/charts/values.yaml.tpl

```diff
@@ -27,7 +28,7 @@ openshift:
     - rhacs-operator
     - {{ $acs.Namespace }}
 {{- end }}
-{{- if $quay.Enabled }}
+{{- if $quayEnabled }}
```

Comment:
All those changes from `quay.Enabled` to `quayEnabled` look wrong to me. Maybe I missed something?

Reply:
ptal https://github.com/redhat-appstudio/rhtap-cli/pull/247#discussion_r1794723328

Reply:
Let's figure out the logic in the other thread, and then see how it impacts this.